scripts/generators/api_dump_generator.py

```diff
@@ -448,7 +448,8 @@ def generate_dispatch_codegen(self):
 
             self.write('if (ApiDumpInstance::current().shouldDumpOutput()) {')
             if command.returnType != 'void':
-                if self.get_unaliased_type(command.returnType) in self.vulkan_defined_types:
+                return_type = self.get_unaliased_type(command.returnType)
+                if return_type in self.vulkan_defined_types or return_type == 'VkDeviceAddress':
```

Comment:
This only affects return values, do you want VkDeviceAddress to be printed as hex in only that case? Or are there just not any places where it is currently used in a struct?

Reply:
aww good call, ya, we use them in `VK_EXT_device_generated_commands` (and many other spots)

will take a version 2 stab at it later today/tomorrow... regardless, was happy it was easy to do

Reply:
updated everywhere

before

![image](https://github.com/user-attachments/assets/d51167a9-7b2b-4aec-aa65-dd6540411a07)

after

![image](https://github.com/user-attachments/assets/3ea89877-a245-43d5-82a4-95e021f9ef48)